src/components/custom-antd/Skeleton/Skeleton.styled.tsx

```diff
@@ -0,0 +1,5 @@
+import {Skeleton} from 'antd';
+
+import styled from 'styled-components';
+
+export const StyledSkeleton = styled(Skeleton)``;
```

Comment:
Could remove the styled skeleton as there is no styling applied

Reply:
We are planning to customize these components in future. For now it is just for a codestyle purpose.

Reply:
We will apply other styles to this component in future. And also it's a good practice to have antd components locally. It will help in case of restyling.